python/processing/algfactory.py

```diff
@@ -208,6 +209,8 @@ def _create_param(self, type, output=False, **kwargs):
             parm = make_func(**kwargs)
             if advanced:
                 parm.setFlags(parm.flags() | QgsProcessingParameterDefinition.FlagAdvanced)
+            if not output:
```

Comment:
No help for outputs?

Reply:
It seems only QgsProcessingParameterDefinition has help